tests/Feature/Http/Controllers/Auth/EmailVerificationTest.php

```diff
@@ -68,7 +68,7 @@ public function testEmailCanBeVerified(): void
         Event::fake();
 
         // Given
-        $user = User::factory()->create([
+        $user = AuthorizedUser::factory()->create([
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create([
````